.github/workflows/WatchCondaForge.yml

```diff
@@ -29,21 +32,35 @@ jobs:
           find: "pangeo-notebook=.*"
           replace: "pangeo-notebook=${{ steps.latest_version.outputs.version }}"
 
+      - name: Check if PR Already Exists
+        id: check_branch
+        run: |
+          # https://github.com/actions/checkout#fetch-all-branches
+          git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
+          if git show-ref --quiet refs/remotes/origin/${PR_BRANCH}; then
+            RESULT=True
+          else
+            RESULT=False
+          fi
+          echo "::set-output name=exists::${RESULT}"
+
       - name: Create Pull Request
+        if: steps.check_branch.outputs.exists == 'False'
```

Comment:
Do we need this line? I understand desire to put this condition on the `condalock` command but it would be nice to keep updating the PR as new packages become available.

Reply:
thinking about this a bit more I think this workflow is better suited to https://github.com/peter-evans/repository-dispatch rather than a cronjob. Since we have control of the metapackage repository we can trigger something here when a new version is released instead of polling every hour? I'll try here https://github.com/scottyhq/githubactions-test and then revisit this PR

Reply:
unfortunately https://github.com/peter-evans/repository-dispatch requires a personal access token, and since we can't add secrets to https://github.com/conda-forge/pangeo-notebook-feedstock that different approach won't work